english/staypuft/ghostbusters/billMurrayMovies/caddyshack/carl-spackler.md

```diff
@@ -0,0 +1,7 @@
+As you near a state of total consciousness, your concentration is broken by
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.